Use (char *) instead of (uint8 *) and fix a spelling error.

--- invaders.orig/video.c
+++ invaders/video.c
@@ -36,7 +36,7 @@
   video_putchar(x+1,y,hextab[hex&15]);
 };
 
-void video_putstring(int32 x, int32 y, uint8 *str)
+void video_putstring(int32 x, int32 y, char *str)
 {
   while (*str) video_putchar(x++,y,*(str++));
 };
--- invaders.orig/video.h
+++ invaders/video.h
@@ -8,7 +8,7 @@
 void video_putchar(int32 x, int32 y, uint8 code);
 void video_fill(int32 x, int32 y, int32 width, int32 height, uint8 code);
 void video_puthex8(int32 x, int32 y, uint8 hex);
-void video_putstring(int32 x, int32 y, uint8 *str);
+void video_putstring(int32 x, int32 y, char *str);
 
 // immediate effect
 void video_update();
--- invaders.orig/game.c
+++ invaders/game.c
@@ -198,7 +198,7 @@
 
   video_usecolor(6,1);
   video_putstring(30,10,"  GAME OVER  ");
-  video_putstring(30,11,winner ? "  YOU WIN  " : "  YOU LOOSE  ");
+  video_putstring(30,11,winner ? "   YOU WIN   " : "  YOU LOSE   ");
   video_putstring(30,12,"  PRESS ESC  ");
   video_update();
   do{
